go-app-ussd.js

```diff
@@ -82,7 +82,8 @@ go.utils = {
             en: 'english',
             fr: 'french',
             sw: 'swahili',
-            so: 'somali'
+            so: 'somali',
```

Comment:
Did this also get updated in `utils.js`? This file is auto-generated and shouldn't be edited directly.